spec/std/big/big_float_spec.cr

```diff
@@ -184,6 +184,18 @@ describe "BigFloat" do
     it { -4321.to_big_f.to_i!.should eq(-4321) }
   end
 
+  describe "to_i64" do
+    it { expect_raises(OverflowError) { (2.0 ** 63).to_big_f.to_i64 } }
+    it { expect_raises(OverflowError) { (BigFloat.new(2.0 ** 63, precision: 128) - 0.9999).to_i64 } }
+    it { expect_raises(OverflowError) { (-9.223372036854778e+18).to_big_f.to_i64 } } # (-(2.0 ** 63)).prev_float
+  end
+
+  describe "to_i64!" do
+    it { (2.0 ** 63).to_big_f.to_i64! }
```

Comment:
Is it not possible to check the expected value for this conversion?

Reply:
The only expectation here is that it doesn't raise. Overflow is intentionally left undefined by GMP even if it returns a value.

Reply:
Kay, maybe leave a comment about that?